iroha/src/peer.rs

```diff
@@ -69,6 +84,8 @@ pub mod isi {
     pub enum PeerInstruction {
         /// Variant of the generic `Add` instruction for `Domain` --> `Peer`.
         AddDomain(String, PeerId),
+        /// Variant of the generic `Register` instruction for `BridgeDefinition` --> `Peer`.
+        RegisterBridge(BridgeDefinition, PeerId),
```

Comment:
I see that for some code you use feature configuration, but here you not use it. Just interesting to know about it.

Reply:
You're right, thank you. I've also fixed other places.